src/components/pages/txs/latest-l1-l2-transactions-table.tsx

```diff
@@ -0,0 +1,106 @@
+// src/components/tx/latest-l1-l2-transactions-table.tsx
+import Link from "next/link";
+import { formatTimestamp } from "@/lib/utils";
+import { Card, CardContent } from "@/components/ui/card";
+import { L1L2Transaction } from "@/lib/types";
+import {
+  Table,
+  TableCaption,
+  TableHeader,
+  TableRow,
+  TableHead,
+  TableBody,
+  TableCell,
+} from "@/components/ui/table";
+import { SquareArrowOutUpRight } from "lucide-react";
+import { l1Chain } from "@/lib/chains";
+
+const LatestL1L2TransactionsTable = ({
+  transactions,
+}: {
+  transactions: L1L2Transaction[];
+}) => (
+  <Card>
+    <CardContent className="p-4">
+      <Table className="w-full table-auto">
+        <TableCaption className="text-lg font-semibold">
+          List of latest L1 - L2 transactions
+        </TableCaption>
+        <TableHeader>
+          <TableRow>
+            <TableHead className="px-4 py-2">Block No</TableHead>
+            <TableHead className="px-4 py-2">L2 Tx Hash</TableHead>
+            <TableHead className="px-4 py-2">Timestamp</TableHead>
+            <TableHead className="px-4 py-2">L1 Tx Hash</TableHead>
+            <TableHead className="px-4 py-2">L1 Tx Origin</TableHead>
+            <TableHead className="px-4 py-2">Gas Limit</TableHead>
+          </TableRow>
+        </TableHeader>
+        <TableBody>
+          {transactions.map((transaction) => (
+            <TableRow key={transaction.l2Hash}>
+              <TableCell className="px-4 py-2 max-w-[10rem] truncate">
+                <div className="flex items-center gap-1 text-sm">
+                  <a
+                    className="flex items-center truncate text-primary hover:brightness-150"
+                    href={`${l1Chain.blockExplorers.default.url}/block/${transaction.l1BlockNumber}`}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    <span className="truncate">{transaction.l1BlockNumber.toString()}</span>
+                    <SquareArrowOutUpRight className="ml-1 w-4 h-4 flex-shrink-0" />
+                  </a>
+                </div>
+              </TableCell>
+              <TableCell className="px-4 py-2 max-w-[14rem] truncate">
+                <div className="flex items-center gap-1 text-sm">
+                  <Link
+                    className="flex items-center truncate text-primary hover:brightness-150"
+                    href={`${l1Chain.blockExplorers.default.url}/tx/${transaction.l2Hash}`}
```

Comment:
The L2 hash is NOT an outside link so simply use `/tx/${transaction.l2Hash}` as href and remove the external icon here.